webapp/src/components/create_issue_menu/index.ts

```diff
@@ -8,8 +8,8 @@ import {isSystemMessage} from 'mattermost-redux/utils/post_utils';
 import {ActionFunc, GenericAction} from 'mattermost-redux/types/actions';
 import { GlobalState } from 'mattermost-redux/types/store';
 
-import {id as pluginId} from '../../manifest';
-import {openCreateIssueModal} from '../../actions';
+import {id as pluginId} from 'src/manifest';
+import {openCreateIssueModal} from 'src/actions';
 
```

Comment:
There's no need for `mapStateToProps` and `mapDispatchToProps` in functional components, so remove the index.ts file for every such component and rename the `component.tsx` file to `index.tsx`.